contracts/RFQ.sol

```diff
@@ -28,9 +30,23 @@ contract RFQ is IRFQ, StrategyBase, ReentrancyGuard, SignatureValidator, BaseLib
         bytes32 transactionHash;
     }
 
-    // Operator events
+    /// @notice Emitted when fee collector address is updated
+    /// @param newFeeCollector The address of the new fee collector
     event SetFeeCollector(address newFeeCollector);
 
+    /// @notice Emitted when order is filled
+    /// @param source The tag of the contract where the order is filled
+    /// @param transactionHash The hash of the transaction structure which is signed by taker
+    /// @param orderHash The hash of the order structure which is signed by maker
```

Comment:
Maybe `which is signed by xxx` could be removed since these are data hash instead of signature.